kubernetes/keycloak/Jenkinsfile

```diff
@@ -0,0 +1,28 @@
+podTemplate(label: 'buildHelm', containers: [
+  containerTemplate(name: 'helm', image: 'lachlanevenson/k8s-helm', ttyEnabled: true, command: 'cat')
+]) {
+   node('buildHelm') {
+     def repo = checkout scm
+     def gitCommit = repo.GIT_COMMIT
+     def gitBranch = repo.GIT_BRANCH
+     def shortGitCommit = "${gitCommit[0..10]}"
+
+    stage('Deploy') {
+        when {
+            expression { return gitBranch == 'master' }
+        }
+        container('helm') {
+            withCredentials([[$class: 'UsernamePasswordMultiBinding', credentialsId: 'keylcloak-admin',
+            usernameVariable: 'KEYCLOAK_USER', passwordVariable: 'KEYCLOAK_PASSWORD']]) {
+                withCredentials([string(credentialsId: 'keycloak-db', variable: 'POSTGRES_HOST')]) {
+                    sh "helm upgrade --install --force --namespace=default \
+                    --values=kubernetes/keycloak/values.yaml \
+                    --set keycloak.persistence.dbHost=${env.POSTGRES_HOST} \
+                    --set keycloak.username=${env.KEYCLOAK_USER} \
+                    --set keycloak.password=${env.KEYCLOAK_PASSWORD} \
```

Comment:
Where are these being set?

Reply:
They are set above in the [credentials block](https://github.com/cfpb/hmda-platform/blob/2af797104cc471d75745cb21f677475c9d4c136d/kubernetes/keycloak/Jenkinsfile#L16) that uses `keycloak-admin` credential.

Reply:
I mean the credentials themselves. I didn't see anything in the `Jenkins` instance related to this